taxonomy-editor-frontend/src/pages/editentry/ListTranslations.jsx

```diff
@@ -74,6 +68,17 @@ const ListTranslations = ({ nodeObject, setNodeObject }) => {
     setIsDialogOpen(false);
   };
 
+  const handleLanguagesChange = (newShownLanguageCodes) => {
```

Comment:
please reanme to something that is more related with the dialog, like `handleDialogConfirm`, otherwise the close dialog action is out of place

Reply:
also, please remove any other `setIsDialogOpen(false);` in the other functions (`handleDeleteTranslation` , etc)